pyomo/contrib/doe/utils.py

```diff
@@ -141,7 +141,7 @@ def compute_FIM_metrics(FIM):
 
     det_FIM : float
         Determinant of the FIM.
-    trace_FIM : float
+    trace_cov : float
```

Comment:
Here again you have replaced `trace_FIM` with `trace_cov` but in most other places you have kept both.